openpathsampling/netcdfplus/dictify.py

```diff
@@ -205,7 +211,7 @@ def to_json(self, obj, base_type=''):
         return json.dumps(simplified)
 
     def to_json_object(self, obj):
-        if hasattr(obj, 'base_cls'):
+        if hasattr(obj, 'base_cls') and type(obj) is not type and type(obj) is not abc.ABCMeta:
```

Comment:
I understood what the code was doing, but I still don't quite understand why. (And I think that, for what it is doing, that's a fine way of doing it). But I don't get what you're saying here:

> This part was not needed before since usually we do not store a class, but rather instances and their attributes. The reference to a class was only necessary to provide a store with the possibility to store for which class this store is meant. This is necessary for all object_stores.

Can you explain what is going on that we are storing classes, as opposed to the attributes of an instance of a given class?